nextflow.config

```
@@ -30,13 +24,24 @@ params {
 
     // Transcript identification method
     assay_type                 = null
+    use_homer_uniqmap          = false
     skip_grohmm                = false
-    skip_tuning                = false
-    tuning_file                = null
+    grohmm_min_uts             = 5
+    grohmm_max_uts             = 45
+    // Depends on how you look at this one... But I figured most will ignore the negative
```

```suggestion
    // Depends on how you look at this one... But I figured most will ignore the negative
```
Could you clarify what you mean here ?

---

Just since it's a negative number, is the min -400 or -100? The docs called it min_ltprobb for groHMM.